packages/jmix-front-generator/src/generators/sdk/all/answers.ts

```diff
@@ -0,0 +1,19 @@
+
+import {StudioProjectInfo} from '../../../common/studio/studio-integration';
+import {ProjectModel} from "../../../common/model/cuba-model";
+import {YeomanGenerator} from "../../../building-blocks/YeomanGenerator";
+import {CommonGenerationOptions} from "../../../common/cli-options";
+import {StudioTemplateProperty} from "../../../common/studio/studio-model";
+
+export interface Answers {
+    projectInfo: StudioProjectInfo;
+}
+
+export const allQuestions: StudioTemplateProperty[] = [];
+
+export const getAnswersFromPrompt = async (
+    projectModel: ProjectModel, gen: YeomanGenerator, options: CommonGenerationOptions
+  ): Promise<Answers> => {
+ 
+    return {} as Answers;
+}
```

Comment:
I think instead of providing a no-op implementation it's better to change the `defaultGetAnswersFromPrompt` so that it does nothing if questions array is empty.

Reply:
@vyacheslav-pushkin
Mayby we will fix it in separate ticket? We have the same situation in App generator. I think it will be better if we will fix this in all generators at once.

Reply:
Agreed